static/images/subscribe-badge-google.svg

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="134px" height="34px" viewBox="0 0 134 34" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 50.2 (55047) - http://www.bohemiancoding.com/sketch -->
+    <title>F00_listen_on_badge</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
```

Comment:
Nit - run this through [SVGOMG](https://jakearchibald.github.io/svgomg/) to remove cruft.

Reply:
I'm surprised Google didn't do this themselves >_<